pennylane/templates/subroutines/qsvt.py

```diff
@@ -621,6 +628,14 @@ def compute_matrix(*args, **kwargs):
         return mat
 
 
+# pylint: disable=protected-access
+if QSVT._primitive is not None:
+
+    @QSVT._primitive.def_impl
+    def _(UA, *projectors, **kwargs):  # kwarg might be id
+        return type.__call__(QSVT, UA, projectors, **kwargs)
```

Comment:
I don't really see a way to avoid this, but the following will likely give an error related to `ArgInfo`:
```python
jax.make_jaxpr(QSVT)(UA, projectors)
```
Because `make_jaxpr` will wrap both args in `ArgInfo` objects 🤦 

This is a silly example, but it could affect other functionality. Consider the following example:
```python
qml.cond(pred, QSVT)(UA, projectors)
```

I wanted to flag this, as it will also affect `qml.qsvt` since the first argument _can_ be a PL operator.

Reply:
Yes, but that constraint is the same as with any of the other nested operators or operator arithmetic.